spec/support/coronavirus_feature_steps.rb

```diff
@@ -4,26 +4,40 @@ def given_i_am_a_coronavirus_editor
 end
 
 def the_payload_contains_the_valid_url
+  live_stream_payload = coronavirus_live_stream_hash.merge(
+    {
+      "video_url" => valid_url,
+      "date" => todays_date,
+    },
+  )
+
   assert_publishing_api_put_content(
-    "774cee22-d896-44c1-a611-e3109cce8eae",
+    coronavirus_content_id,
     request_json_includes(
       "details" => {
         "announcements_label" => "Announcements",
-        "live_stream" => {
-          "video_url" => valid_url,
-          "date" => todays_date,
-          "date_text" => "Live streamed",
-          "previous_videos" => {
-            "url" => "https://www.youtube.com/user/Number10gov/videos",
-          },
-        },
+        "live_stream" => live_stream_payload,
       },
     ),
   )
 end
 
-def stub_publishing_api_live_content_request
-  stub_publishing_api_has_item(JSON.parse(live_content_item))
+def stub_live_coronavirus_content_request
+  stub_publishing_api_has_item(JSON.parse(live_coronavirus_content_item))
+end
+
+def live_coronavirus_content_item
+  File.read(Rails.root.join + "spec/fixtures/coronavirus_content_item.json")
+end
+
+def coronavirus_live_stream_hash
+  h = JSON.parse(live_coronavirus_content_item)
+  h["details"]["live_stream"]
+end
+
+def coronavirus_content_id
+  h = JSON.parse(live_coronavirus_content_item)
```

Comment:
That the third repetition of `JSON.parse(live_coronavirus_content_item)`. I'd suggest
```ruby
def coronavirus_content_json
  @coronavirus_content_json ||= JSON.parse(live_coronavirus_content_item)
end
```
And then update the three instances to use that method. So here:
```ruby
def coronavirus_content_id
  coronavirus_content_json["content_id"]
end
```